marketplace-elements.js

```diff
@@ -404,6 +404,26 @@
         }),
     });
 
+    document.registerElement('mkt-section', {
+        prototype: Object.create(MktHTMLElement.prototype, {
+            createdCallback: {
+                value: function() {
+                    this.classList.add('mkt-section');
+
+                    if (this.hasAttribute('full')) {
```

Comment:
Thinking out loud (not suggesting you make this change), it might be nice to have something like this:

```js
this.handleAttribute('full', {
  addClass: 'mkt-section-full',
  added: function() {
    var div = document.createElement('div');
    forEach(this.children, function(child) {
      div.appendChild(child);
    });
    this.appendChild(div);
  },
  removed: function() {
    var div = this.children[0];  // Might want a better check.
    forEach(div.children, function(child) {
      this.appendChild(child);
    });
    this.removeChild(div);
  },
});
```